src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,14 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: all 300ms;
```

Comment:
no. since we have only two changed properties, list them individually.

Reply:
done